test/snippet/utility/bloom_filter/bloom_filter_contains.cpp

```diff
@@ -0,0 +1,15 @@
+#include <seqan3/core/debug_stream.hpp>
+#include <seqan3/utility/bloom_filter/bloom_filter.hpp>
+
+int main()
+{
+    seqan3::bloom_filter bf{seqan3::bin_size{1024u}};
+    bf.emplace(126);
+    bf.emplace(712);
+    bf.emplace(237);
+
+    // Query the Bloom Filter. Note that there may be false positive results!
+    // A return of `true` indicates the (probable) presence of the query in the Bloom Filter.
```

Comment:
```suggestion
    // A return of `true` indicates the (probable) presence of the query in the Bloom Filter.
```

Isn't the real strong point of a bloom filter is that a `false` definitely means not contained? And a `true` might actually be a `false`?

Reply:
To be honest, I think I just copied this from the IBF snippet.  
Probably, we could make this a bit more clear. However, it's some more comment lines:

```c++
// Query the Bloom Filter. 
// A return of `false` guarantees the query not being present in the Bloom Filter.
// A return of `true` indicates the (probable) presence of the query in the Bloom Filter. 
// Note that there may be false positive results!
```

Reply:
I think it is ok to be more clear about it.